src/Components/Publishing/Header/FeatureHeader.tsx

```diff
@@ -190,74 +229,80 @@ const TextAsset = styled.div`
 `
 const SubHeader = styled.div`
   ${Fonts.unica("s19", "medium")}
-  display: flex;
+      display: flex;
   justify-content: space-between;
   align-items: flex-end;
   flex-direction: row;
-  ${pMedia.sm`
+
+  ${breakpoint.sm`
     align-items: flex-start;
     flex-direction: column;
   `}
-`
+      `
 const Title = styled.div`
   ${Fonts.unica("s100")}
-  margin-bottom: 75px;
+      margin-bottom: 75px;
   letter-spaceing: -0.035em;
-  ${pMedia.xl`
+
+  ${breakpoint.xl`
     ${Fonts.unica("s80")}
   `}
-  ${pMedia.md`
+
+  ${breakpoint.md`
     ${Fonts.unica("s65")}
   `}
-  ${pMedia.xs`
+
+  ${breakpoint.xs`
     ${Fonts.unica("s45")}
   `}
 `
 const Deck = styled.div`
   max-width: 460px;
   margin-right: 30px;
-  ${Fonts.unica("s16", "medium")}
   line-height: 1.4em;
-  ${pMedia.sm`
+
+  ${Fonts.unica("s16", "medium")}
+
+      ${breakpoint.sm`
     margin-bottom: 28px;
     ${Fonts.unica("s14", "medium")}
   `}
-`
+      `
 const FeatureHeaderContainer = Div.extend`
   width: 100%;
   height: ${props => props.height};
   &[data-type="text"] {
     height: auto;
     ${Title} {
-      margin-bottom: 150px;
+        margin - bottom: 150px;
     }
   }
   &[data-type="split"] {
-    ${Title} {
-      flex-grow: 1;
+        ${Title} {
+        flex - grow: 1;
     }
     ${HeaderText} {
-      width: 50%;
+        width: 50%;
     }
     ${FeatureImage} {
-      width: 50%;
+        width: 50%;
       border: 20px solid white;
     }
     ${FeatureVideoContainer} {
-      width: 50%;
+        width: 50%;
       border: 20px solid white;
     }
     ${FeatureVideo} {
-      width: 50vw;
+        width: 50vw;
     }
     ${SubHeader} {
-      align-items: flex-start;
+        align - items: flex-start;
       flex-direction: column;
     }
     ${Deck} {
-      margin-bottom: 30px;
+        margin - bottom: 30px;
```

Comment:
@alloy, @orta - this is what I mean by bugs in Prettier; all of the formatting was blown up and now its introducing literal errors in our styles. Not sure why this is happening but would like to disable so as not to slow us down in this sprint.

Reply:
Prettier can probably have it's inline CSS styling disabled somehow

Reply:
Looks like various issues around this were introduced in v1.5.0 https://github.com/prettier/prettier/issues/2291, https://github.com/prettier/prettier/issues/2623, https://github.com/prettier/prettier/issues/2350

I haven’t found any issue yet around nested rules, so that that’s a ticket that definitely needs to get filed with them.

I wonder if in the interim there’s another fix we can apply. E.g. the places were the linebreak after interpolated CSS gets removed can be fixed by adding an extra semicolon after the interpolation:

```diff
 const Vertical = styled.div`
-  ${Fonts.unica("s16", "medium")}
+  ${Fonts.unica("s16", "medium")};
   margin-bottom: 10px;
   ${breakpoint.sm`
-    ${Fonts.unica("s14", "medium")}
+    ${Fonts.unica("s14", "medium")};
-  `}
+  `};
 `
```

Reply:
Hmm, this is weird, I can’t actually reproduce this specific breakage on save that you highlight @damassi, only the ones that I could fix by adding semicolons. Any chance there are other vscode extensions at play here?

Reply:
For examples see [this playground snippet](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEaIFs4AiAlgG6YC82MAngDZwAmAdA6QAYA6UmmAJMBxAwAFkTSYARgCc4AQwDWaQQF8uPSLQhSkmGQwDcXTlC6QoGTBBFwpxMpQx1GLdmr7Ac+O8swDuPTFxZKQBzIigAWgkrPB0AJgAGAAcAD0N-VSg2EAAaEAgkmCJ0ZFBZDGQAM1laNDg86VkweTgYAGUkpvCQ5BgpAFd6kAYIMCqauryAKzQUgCEpJpb22XwAGXC4cdqhiH6YJP247cmQTqk6qWQQCVkJOmhcs6lwmAB1IgYRZAAOBLy6rgiL0BkM0N16ABFfpWLZIao7PIwO4fL7CZBxJGLIi0boAYTwQWuUGgWzy-TqABU7mgTnBlMogA).

Reply:
@orta From those tickets I linked to it seems like you can only disable prettier completely (with disable comments).

Reply:
Hrm, good point (you can only disable it [inside vscode](https://github.com/prettier/prettier-vscode/#prettierstylelintintegration-default-false---css-scss-and-less-only) ) in which case it'd mysteriously get clobbered on commit.

Reply:
@alloy, @orta - yeah, i'm not too sure what the deal is and def don't have time to debug right now... 

In the meantime, yet another issue I just caught; check out this screen cap. On save its munging the spacing: 

![prettier](https://user-images.githubusercontent.com/236943/31906966-5867f954-b7e7-11e7-9d93-6f96ca9dbabf.gif)

So (so) annoying... On commit this tool can do what it will, but while I'm writing code I'm pretty sure this stuff shouldn't be taking away from a developers focus. I feel kinda strongly about this. If a developer wants Prettier and TSLint-fix to reformat their code while they're typing they can very easily opt in via their personal editor settings, but it shouldn't be forced on us.

Reply:
I don't get why it's so buggy in Reaction? It's been perfect in Emission and Danger

Reply:
The last time people were having problems with this it was because they had other extensions installed which were breaking it on save

Reply:
👍 Just spoke to @alloy and I'm gonna dig into my editor and check to see if there's a conflict somewhere, which seems likely. Thanks y'all

Reply:
Let us know if you find it, I’m so dumbfounded otherwise.

Reply:
Found the culprit: https://marketplace.visualstudio.com/items?itemName=mgmcdermott.vscode-language-babel. Disabled and all ok!

Reply:
Interesting, I wonder if it's a language scoping issue ( e.g. the template string css is classed as something else )

Reply:
Glad you found it!

Weird that the README says it’s _only_ syntax highlighting, but changes how code is formatted on save, though 🤔